product_brand/views/product_brand_view.xml

```diff
@@ -165,10 +165,7 @@
         <field name="model">product.product</field>
         <field name="inherit_id" ref="product.product_kanban_view" />
         <field name="arch" type="xml">
-            <xpath
-                expr="//div[hasclass('oe_kanban_details')]//strong[1]"
-                position="after"
-            >
+            <xpath expr="//div[hasclass('d-flex')]" position="after">
```

Comment:
Tested in runboat: when I click on the brand in the kanban view of a variant (product.product), the link doesn't work because it opens directly the variant instead of the brand. Could you check? (but seems working fine on the product.template)

Also here, avoid to do generic search on class `d-flex`.

Reply:
It doesn't work on v17 either. I fixed it by adding the widget  `widget="many2one"` .